### ISSUE TITLE
Post-cms-Freeze-UAT-Drupal

### ISSUE THREAD

**Issue by Karak888** [department-of-veterans-affairs/va.gov-team#1020]
## High Level User Story
As a _____, I need _____ so I can _____.

## Hypothesis/(es)
**If** _we make this change_ **then** _we expect this to happen_.

## Goals
_What outcome(s) do we want to see?_ 

## Objectives
_Which VSP Objectives does this body of work push forward?_ (`Intake & Onboarding`, `Automation + Instrumentation`, `Support`, `Capacity-building`)

## Definition of Done
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with `Epic`**

**Comment by jenniferlee-dsva** [comment]
This old ticket for Special claims page (in Disability) also needs to be made during post-UAT work: 
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15832#issuecomment-530198327

**Comment by Karak888** [comment]
github pages that are live now are out of sync with drupal insttance this ticket is to ensure all emergency updates needed to be made in github instance get made to the drupal instance comes out of UAT.   Last remaining hubs in UAT:  eduacation, burials, disability, family and service members hubs.

**Comment by Karak888** [comment]
@RLHecht are we still waiting on the following issue:  https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15832#issuecomment-530198327 

Thanks!

**Comment by RLHecht** [comment]
@Karak888 Yes, this still needs to be written, reviewed and put in. Jen said it can wait till after the page goes live so it does not have to be done right away.
